src/markdown.js

```diff
@@ -67,17 +102,24 @@ export default function (config, options = {}) {
 				return defaultRender(tokens, idx, options, env, self).replace(...epubRules[rule]);
 			}
 		});
-		// md.renderer.rules.footnote_ref = function (tokens, idx, options, env, slf) {
-		// 	const id = slf.rules.footnote_anchor_name(tokens, idx, options, env, slf);
-		// 	const caption = slf.rules.footnote_caption(tokens, idx, options, env, slf);
-		// 	let refid = id;
 
-		// 	if (tokens[idx].meta.subId > 0) {
-		// 		refid += `:${tokens[idx].meta.subId}`
-		// 	}
+		md.renderer.rules.footnote_ref = function (tokens, idx, options, env, slf) {
+			let id = slf.rules.footnote_anchor_name(tokens, idx, options, env, slf);
+			let caption = slf.rules.footnote_caption(tokens, idx, options, env, slf);
+
+			let refId = id;
+			if (tokens[idx].meta.subId > 0) {
+				refId += `:${tokens[idx].meta.subId}`;
```

Comment:
What does that do and why?

Reply:
Actually, I don't know. This code is a copy of the footnotes plugin with the necessary changes. So... I don't know what to write in the comment 🙈

Reply:
You write exactly that. "// Code from <link to code on github>". But do we really need to copy? Maybe we can store a reference to the existing function and monkey patch? Once I have that link I can tell you

Reply:
Fair enough. Done!

Reply:
Looks like we can wrap the function and just append to its return value?

Reply:
Oh, good idea! I need to try this out